packages/devextreme/js/ui/rating.d.ts

```diff
@@ -0,0 +1,186 @@
+import {
+    EventInfo,
+    NativeEventInfo,
+    InitializedEventInfo,
+    ChangedOptionInfo,
+} from '../events/index';
+
+import Editor, {
+    EditorOptions,
+    ValueChangedInfo,
+} from './editor/editor';
+
+/** @public */
+export type RatingSelectionMode = 'continuous' | 'single';
+
+/**
+ * @docid _ui_rating_ContentReadyEvent
+ * @public
+ * @type object
+ * @inherits EventInfo
+ */
+export type ContentReadyEvent = EventInfo<dxRating>;
+
+/**
+ * @docid _ui_rating_DisposingEvent
+ * @public
+ * @type object
+ * @inherits EventInfo
+ */
+export type DisposingEvent = EventInfo<dxRating>;
+
+/**
+ * @docid _ui_rating_InitializedEvent
+ * @public
+ * @type object
+ * @inherits InitializedEventInfo
+ */
+export type InitializedEvent = InitializedEventInfo<dxRating>;
+
+/**
+ * @docid _ui_rating_OptionChangedEvent
+ * @public
+ * @type object
+ * @inherits EventInfo,ChangedOptionInfo
+ */
+export type OptionChangedEvent = EventInfo<dxRating> & ChangedOptionInfo;
+
+/**
+ * @docid _ui_rating_ValueChangedEvent
+ * @public
+ * @type object
+ * @inherits NativeEventInfo,ValueChangedInfo
+ */
+export type ValueChangedEvent = NativeEventInfo<dxRating, KeyboardEvent | MouseEvent | PointerEvent | TouchEvent | Event> & ValueChangedInfo;
+
+/**
+ * @public
+ * @namespace DevExpress.ui
+ * @deprecated use Properties instead
+ * @docid
+ */
+export interface dxRatingOptions extends EditorOptions<dxRating> {
+    /**
+     * @docid
+     * @default true
+     * @public
+     */
+    activeStateEnabled?: boolean;
+    /**
+     * @docid
+     * @default true
+     * @public
+     */
+    focusStateEnabled?: boolean;
+    /**
+     * @docid
+     * @default true
+     * @public
+     */
+    hoverStateEnabled?: boolean;
+    /**
+     * @docid
+     * @default 5
+     * @public
+     */
+    itemCount?: number;
+    /**
+     * @docid
+     * @default {}
+     * @public
+     */
+    label?: {
+        /**
+        * @docid
+        * @default false
+        */
+        enabled?: boolean;
+       /**
+        * @docid
+        * @default function(value, itemCount) { return `${value}/${itemCount}` }
+        */
+       format?: string | ((value: number, itemCount: number) => string);
```

Comment:
I think we should double-check this type, because initially, we intended to use the existing Format type. If we make this unique, our tech writers should describe it in the documentation. Please add a separate task to discuss, improve, and describe it.

Reply:
Please remove this property for now because it's still questionable. U can leave only `enabled` here for the moment